content/blog/esc-webhooks-launch/index.md

```diff
@@ -0,0 +1,58 @@
+meta_image: "meta.png"
+authors:
+  - iaroslav-titov
+  - arun-loganathan
+tags:
+  - esc
+  - secrets
+  - features
+---
+
+Managing secrets and configurations across multiple environments and teams can be a complex juggling act for development teams. [Pulumi ESC](/docs/esc/), our developer-friendly secrets management product, simplifies this process and accelerates your development cycle. Today, we're excited to make this process even easier with the launch of Webhooks for Pulumi ESC, a powerful new feature that allows you to send a custom trigger to any URL endpoint, enabling you to automate workflows and keep your infrastructure and applications up to date. Pulumi ESC Webhooks also integrates seamlessly with popular communication platforms like Slack and Microsoft Teams.
+
+<!--more-->
+
+With Pulumi ESC Webhooks, you can now receive real-time notifications and trigger automated actions based on specific events within your Pulumi ESC [Environments](/docs/esc/environments/).  For example, you can set up a Webhook to notify your team’s Slack channel whenever a new secret is added to an environment, or automatically trigger a redeployment pipeline when configurations are changed. Webhooks can be triggered by changes to [version tags](/docs/esc/environments/#tagging-versions) and [environment tags](/blog/esc-projects-environment-tags-launch/#introducing-environment-tags), and can be configured per environment or at the organization level for all environments.
+
+Take your automation further by creating Webhooks that trigger [Pulumi Deployments](/docs/pulumi-cloud/deployments/), automatically updating your infrastructure in response to configuration changes in Pulumi ESC—without writing custom code and all within the Pulumi Cloud Console.
+
+Hierarchical configurations where one environment [imports](/docs/esc/environments/#importing-other-environments) others are fundamental to Pulumi ESC. If an environment is importing another environment, any changes made to the imported environment are also received as a Webhook configured within the current environment. The Webhook will notify you that the environment has changed, and you can call back to open the environment using the REST API, Pulumi Service Provider, Automation API, and SDK, and get the latest secrets and configuration.
+
+## Getting Started with Pulumi ESC Webhooks
+
+Setting up Webhooks is straightforward. You can use the Pulumi Cloud Console, the ESC SDK, the Pulumi Service Provider, or REST API.
+
+### Using Pulumi Cloud Console
+
+- Navigate to the "Webhooks" tab under settings for organization-wide Webhooks or within an individual environment's top navigation bar.
+- Select your desired integration (Slack, Microsoft Teams, or generic JSON Webhooks).
+- Configure the Webhook details, including the events you want to trigger notifications on.
+
+<b>(Add a video demo/image)</b>
+
+### Using Pulumi Service Provider
+
+<b>(To be added)</b>
+
+### Using the SDK
+
+<b>(To be added)</b>
```

Comment:
This one https://github.com/pulumi/pulumi-service/issues/20923 was a could have and we probably won't have it